include/treelite/tree_impl.h

```diff
@@ -627,7 +631,8 @@ inline void
 Tree<ThresholdType, LeafOutputType>::Init() {
   num_nodes = 1;
   leaf_vector_.Clear();
-  leaf_vector_offset_.Resize(2, 0);
+  sleaf_vector_begin_.Resize(1, {});
```

Comment:
```suggestion
  leaf_vector_begin_.Resize(1, {});
```
Stray `s`?

Reply:
Whoops, I forgot to commit the last changes.

Reply:
Any updates on this one?

Reply:
Fixed on my end.